src/sentry/auth/idpmigration.py

```diff
@@ -55,8 +59,14 @@ def create_verification_key(user: User, org: Organization, email: str) -> None:
 
     send_confirm_email(user, email, verification_code)
 
+    return verification_code
 
-def verify_new_identity(user: User, org: Organization, key: str) -> bool:
+
+def get_redis_key(verification_key: str) -> str:
+    return f"auth:one-time-key:{verification_key}"
+
+
+def verify_new_identity(key: str) -> bool:
```

Comment:
`verify_account`? since we aren't verifying the new identity here, but that the user owns their account